_dev/src/components/product-feed/product-feed-settings.vue

```diff
@@ -78,13 +80,14 @@ export default {
     SyncSchedule,
     Summary,
   },
-  data() {
-    return {
-    };
-  },
   computed: {
     activeStep() {
-      return this.$store.state.productFeed.stepper;
+      if (this.$store.state.productFeed.stepper > 0) {
+        return this.$store.state.productFeed.stepper;
+      }
+      const stepIs = ProductFeedSettingsSteps.indexOf(this.$route.params.step);
```

Comment:
I think you need to check indexOf return, he can give you -1 if he does not find it